source/includes/api/core/_data_widgets.md

```diff
@@ -0,0 +1,21 @@
+## Data Widgets
+
+### Get all a Programme's Data Widgets
+
+To list all data_widgets that belong to a programme (scoped by api key)
+
+> Header:
+
+``` http
+GET /api/v2/data_widgets HTTP/1.1
+Authorization: Token token=xxx
+```
+
```

Comment:
missing the response tag here... refer to the kittens example

https://github.com/CorporateRewards/docs-myrewards/blob/master/source/includes/example/_kittens.md